examples/python/client_http_no_middleware.py

```diff
@@ -178,7 +146,6 @@ def signal_handler(sig: int, frame: Optional[FrameType]) -> None:
         traceback.print_exc()
         print(f"Failed to create client instance ({ex})")
     finally:
-        results_viewer.stop()
```

Comment:
Are you sure about that? How does the viewer exit if the entire video is read and the program is not exited with Ctrl+C or kill?

Reply:
The ResultsViewer is a daemon thread here, so it should not be necessary to stop it (it will be terminated automatically at the end of the program). If we decide to change it to non-daemonic thread, then yes, we would need to stop it like this.